[2024-12-19T05:42:02.633+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T05:42:02.642+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2023-04-27T00:00:00+00:00 [queued]>
[2024-12-19T05:42:02.647+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2023-04-27T00:00:00+00:00 [queued]>
[2024-12-19T05:42:02.647+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T05:42:02.662+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2023-04-27 00:00:00+00:00
[2024-12-19T05:42:02.665+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2851) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T05:42:02.665+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2023-04-27T00:00:00+00:00', '--job-id', '641', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpbv36x1ls']
[2024-12-19T05:42:02.666+0000] {standard_task_runner.py:72} INFO - Started process 2856 to run task
[2024-12-19T05:42:02.666+0000] {standard_task_runner.py:105} INFO - Job 641: Subtask first_task
[2024-12-19T05:42:02.690+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2023-04-27T00:00:00+00:00 [running]> on host 13c961a72918
[2024-12-19T05:42:02.730+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='code2j' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2023-04-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-27T00:00:00+00:00'
[2024-12-19T05:42:02.731+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T05:42:02.739+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T05:42:02.740+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2024-12-19T05:42:02.745+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T05:42:02.747+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2024-12-19T05:42:02.747+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T05:42:02.762+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T05:42:02.763+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2023-04-27T00:00:00+00:00, execution_date=20230427T000000, start_date=20241219T054202, end_date=20241219T054202
[2024-12-19T05:42:02.805+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T05:42:02.813+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T05:42:02.814+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
